matter_server/server/device_controller.py

```diff
@@ -682,14 +682,12 @@ async def remove_node(self, node_id: int) -> None:
             return
         result: Clusters.OperationalCredentials.Commands.NOCResponse | None = None
         try:
-            result = await self._call_sdk(
-                self.chip_controller.SendCommand(
-                    nodeid=node_id,
-                    endpoint=0,
-                    payload=Clusters.OperationalCredentials.Commands.RemoveFabric(
-                        fabricIndex=fabric_index,
-                    ),
-                )
+            result = await self.chip_controller.SendCommand(
```

Comment:
So is SendCommand safe to call from a different Thread?

Reply:
Well, that is the big mystery for all these async calls. My current hunch is that it is as its called from the mainthread.

Reply:
Ah I see this is an async one. I mean we call it from (our) main thread. The SendCommand then uses `src/controller/python/chip/ChipStack.py:ChipStack Call`, it uses thread safe locks (the networkLock) and `PostTaskOnChipThread` to run the task on the chip Thread. So I think it is fairly safe to assume that we can run this from any Thread. :+1: